src/main/java/com/hanzereversiai/projectp3/networking/Network.java

```diff
@@ -0,0 +1,30 @@
+package com.hanzereversiai.projectp3.networking;
+
+import java.io.*;
+import java.net.InetSocketAddress;
+import java.net.Socket;
+import java.net.SocketAddress;
+
+public class Network {
+
+    public String hostname = "127.0.0.1";
+    public int port = 7789;
+    public int timeout = 30;
+
+    private Connection connection;
+
+    public Network() {
```

Comment:
Please add a second constructor where hostname, port and timeout are parameters.